test/infrastructure/docker/main.go

```diff
@@ -135,6 +136,9 @@ func initFlags(fs *pflag.FlagSet) {
 	fs.IntVar(&concurrency, "concurrency", 10,
 		"The number of docker machines to process simultaneously")
 
+	fs.IntVar(&clusterCacheTrackerConcurrency, "clustercachetracker-concurrency", 10,
```

Comment:
If this isn't set, could we actually default to `concurrency`? It makes some sense to keep these in sync

Reply:
I think it doesn't make much sense to keep them in sync. Usually you need more workers for a controller when they otherwise can't keep up with the load. This mainly depends on reconcile duration. 

Looking at the Reconcile func of ClusterCacheReconciler, it will always return almost instantly. So even with 10k> Clusters I don't think we need more than 10 workers.

In fact the only reconcile that I had to run with more than 10 works with 2k clusters was KubeadmControlPlane (because it had reconcile durations of ~ 1-2 seconds).